entities/library.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require_relative '../dependencies'
+
+class Library
+  include Database
+  include Statistics
+
+  attr_reader :authors, :books, :readers, :orders
+
+  def initialize (authors: [], books: [], readers: [], orders: [])
+    @authors = authors
+    @books = books
+    @readers = readers
+    @orders = orders
+    load_data
+  end
+
+  def delete_file
+    delete_data
```

Comment:
remove delete_file method
`delete_data` should be called on the library instance